dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -164,4 +165,129 @@ def test_case_sensitivity(self):
             expected_word_count = {'hello': 1, 'name': 1, 'grant': 2, 'bob': 1,
                                    'friends': 1}
             self.assertDictEqual(expected_word_count, profile['word_count'])
+
```

Comment:
we need tests validating setting the options work, and they should be in their own tests like the other options
We can keep the integration tests which validate they options affect things how we would expect them here.

Reply:
I would also suggest making these tests smaller as each test should limit to a specific scope